.github/workflows/Dependencies Security Scan.yml

```diff
@@ -31,17 +32,8 @@ jobs:
       fail-fast: false
 
     steps:
-      - name: Checkout code
-        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b  # v3.0.2
       - name: Checkout Ed-Fi-ODS
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
-        with:
-            repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
-            path: Ed-Fi-ODS/
-      - name: Is pull request branch exists in Ed-Fi-ODS
-        working-directory: ./Ed-Fi-ODS/
-        shell: pwsh
-        run: |
-             .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "."
```

Comment:
Dependency review also runs in the Ed-Fi-ODS-Implementation repository, in its own workflow. Therefore we don't need _this_ job to look at both repositories: it is safe to only look at the current repository.